plugins/HostWindows.cpp

```diff
@@ -49,7 +50,8 @@ Modules HostWindows::modules() const {
 		MEMORY_BASIC_INFORMATION64 mbi;
 		auto address = reinterpret_cast< procptr_t >(me.modBaseAddr);
 		while (VirtualQueryEx(processHandle, reinterpret_cast< LPCVOID >(address),
-							  reinterpret_cast< PMEMORY_BASIC_INFORMATION >(&mbi), sizeof(mbi))) {
+							  reinterpret_cast< PMEMORY_BASIC_INFORMATION >(&mbi), sizeof(mbi))
+			   && (mbi.AllocationBase == reinterpret_cast< procptr_t >(me.modBaseAddr))) {
```

Comment:
There should be a comment explaining what is done here and why. That way future coders have an easier time understanding what's happening :)